scripts/main.py

```diff
@@ -36,23 +38,40 @@ def extract_feature_vectors(TRAINING_SIZE, data_dir, train_data_filename):
         print(image_filename)
         if os.path.isfile(image_filename):
             img = Image.open(image_filename)
-
             t_img = to_tensor(img).unsqueeze(0) # 1 x 3 [rgb] x 400 x 400
-
             imgs.append(t_img)
-    return imgs # length TRAINING_SIZE
+
+            if (rotate == True):
+                width, height = img.size
+                # Rotate images
+
+                for degree in range(360):
+                    rotated_img = img.rotate(degree)
+                    # Crop to remove black parts
+                    left     = width / 4 * 0.58
+                    top      = height / 4 * 0.58
```

Comment:
This computation was made for the worst case scenario  (in terms of # back pixels in the rotated image which happens at 45 degree angle)